index.html

```diff
@@ -4,18 +4,47 @@
     <title>Tic Tac Toe!</title>
     <meta charset="utf-8">
     <link href="index.css" media="screen" rel="stylesheet" type="text/css">
+    <link href='https://fonts.googleapis.com/css?family=Patrick+Hand+SC' rel='stylesheet' type='text/css'>
   </head>
 
   <body>
-    <h1>Tic Tac Toe</h1>
-    <div id="tic-tac-toe"></div>
+    <h1>Tic-Tac-Toe</h1>
+
+    <button class="play-button">PLAY!</button><br>
+    <div class="play">
+      Player 1
+      <p class="emoji">🐱</p>
+    </div>
+    <div class="play">
+      Player 2
+      <p class="emoji">🐶</p>
+    </div>
+    <div class="current-turn">Current Turn: <span class="player-turn"></span><div>
+    <div class="message-cat"></div>
+    <div class="message-dog"></div>
+    <div id="tic-tac-toe">
+    <table>
+    <tr>
+      <td><button class="square a" data-square="a"></button></td> 
+      <td><button class="square b" data-square="b"></button></td>   
+      <td><button class="square c" data-square="c"></button></td>
+    </tr>
+    <tr>
+      <td><button class="square d" data-square="d"></button></td> 
+      <td><button class="square e" data-square="e"></button></td>   
+      <td><button class="square f" data-square="f"></button></td>
+    </tr>
+    <tr>
+      <td><button class="square g" data-square="g"></button></td> 
+      <td><button class="square h" data-square="h"></button></td>   
+      <td><button class="square i" data-square="i"></button></td>
+    </tr>
```

Comment:
I understand the idea behind naming these letters, but possibly a description of where they are would help? like column 1, row 1 (c1r1) for descriptive purposes? Just a thought